src/monodroid/jni/osbridge.hh

```diff
@@ -117,7 +117,9 @@ namespace xamarin::android::internal
 		void initialize_on_onload (JavaVM *vm, JNIEnv *env);
 		void initialize_on_runtime_init (JNIEnv *env, jclass runtimeClass);
 		void add_monodroid_domain (MonoDomain *domain);
+#if !defined (NET6)
```

Comment:
What this brings to mind is that `Runtime.destroyContexts()` is "bad" ("weird"?), in that it is a Java method which exists "everywhere", but is only implemented on Desktop.  I assume that if anyone tried to invoke this on Android they'd get an exception, as `monodroid-glue-designer.cc` isn't part of the non-Desktop build.

:-(